changelog/@unreleased/pr-472.v2.yml

```
@@ -0,0 +1,5 @@
+type: feature
+feature:
```

Given that we haven't moved _everything_ to jdk17+ quite yet, let's call this a break and major rev, so folks don't get confused (version numbers are cheap!). Worth updating the changelog to note the jdk17 requirement since this is fairly widely used.
```suggestion
type: break
break:
```